<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vec3Editor</class>
 <widget class="QWidget" name="Vec3Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LineEditDouble" name="xLineEdit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LineEditDouble" name="yLineEdit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LineEditDouble" name="zLineEdit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditDouble</class>
   <extends>QLineEdit</extends>
   <header>LineEditDouble.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
